### PR TITLE
[docs] add Grafana integration guide page

### LINKED ISSUE #8275: [Docs]: Grafana docs integration guide
### Short description of what should be added or improved

Creating an integration guide between MindsDB and  [Grafana ](https://grafana.com/)(an open source BI)

### Video or screenshots

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by maulanaakbardj**
## Description

This PR addresses this issue:

Fixes #8275

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] 📄 This change requires a documentation update

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

pull1

pull2

### CODE REVIEW

File: docs/connect/grafana.mdx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+Click on the `Save & test` button to check if all the provided data
+allows you to connect to MindsDB.
+
+On success, you should see the message, as below.
+
+<p align="center">
+  <img src="/assets/sql/grafana_3.png" />
+</p>
+
+## Let's Run Some Queries
```
Comment (martyna-mindsdb) [inline docs/connect/grafana.mdx:74]: Let's call this section `## Examples`

Then, add two subsections:
1. `### Querying` should include querying for full databases
2. `### Visual Query Builder` should include its content as is

**Comment by martyna-mindsdb** [comment]
@maulanaakbardj 
Please see the review comments above.

File: docs/connect/grafana.mdx [hunk 2]
```diff
@@ -0,0 +1,134 @@
+---
+title: MindsDB and Grafana
+sidebarTitle: Grafana
+---
+
+[Grafana](https://grafana.com/) is an open-source analytics and interactive visualization web application
+that allows users to ingest data from various sources, query this data, and display it on customizable charts for easy analysis. 
+
+## Data Source Grafana Setup
```
[the remark this answers is not in the document]
Comment (maulanaakbardj) [reply docs/connect/grafana.mdx]: done

File: docs/connect/grafana.mdx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+First, Go to the `Connections` section click on `Add new connection` and Choose the `MySQL` plugin
```
[the remark this answers is not in the document]
Comment (maulanaakbardj) [reply docs/connect/grafana.mdx]: done

File: docs/connect/grafana.mdx (hunk 1 above)
Comment (maulanaakbardj) [reply docs/connect/grafana.mdx]: done

**Comment by maulanaakbardj** [comment]
> @maulanaakbardj Please see the review comments above.

@martyna-mindsdb , I've made an update based on your review comments

**Review by martyna-mindsdb — APPROVED** [review]

**Pull request merged**